coalib/bearlib/abstractions/Linter.py

```diff
@@ -96,6 +96,9 @@ def _prepare_options(options):
         raise ValueError("Invalid `output_format` specified.")
 
     if options["prerequisite_check_command"]:
+        logging.warn("The bear which uses the following executable --{} "
+                     "uses ``prerequisite_check_command`` which is "
+                     "depreciated.".format(options["executable"]))
```

Comment:
and second: You should print the name of the bear directly instead of "The bear which uses the following executable -- ..."
(especially why do you have those `--`?)